roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -366,3 +365,6 @@ etcd_events_peer_addresses: |-
   {%- endfor %}
 
 podsecuritypolicy_enabled: false
+etcd_heartbeat_interval: "250"
+etcd_election_timeout: "5000"
```

Comment:
These are never used ?

Reply:
They used following line:

https://github.com/kubernetes-incubator/kubespray/blob/f5f7b1626b3079e864e09a49b128e90e95d1ff77/roles/etcd/templates/etcd.env.j2#L9

Is that wrong?